src/routes/stake/Stake.svelte

```diff
@@ -202,21 +199,23 @@
 
 	.paragraphs {
 		display: flex;
-		justify-content: space-around;
 		flex-direction: column;
-		height: 8em;
 		position: relative;
+		gap: 1em;
 	}
 
 	.error {
-		color: red;
+		display: flex;
+		align-items: center;
+		color: var(--title-color);
+		gap: 10px;
```

Comment:
I think this is too much on mobile, 
```suggestion
		gap: 0.2em;
```

Reply:
done